app/scripts/controllers/client/ViewClientCollateralController.js

```diff
@@ -0,0 +1,59 @@
+(function (module) {
+    mifosX.controllers = _.extend(module, {
+        ViewClientCollateralController: function (scope, resourceFactory, routeParams, location, $uibModal) {
+
+            scope.formData = {};
+            scope.clientId = routeParams.id;
+            scope.collateralId = routeParams.collateralId;
+            scope.loanTransactions = [];
+            console.log(routeParams);
+
+            resourceFactory.clientcollateralResource.get({clientId: scope.clientId, collateralParamId: scope.collateralId}, function (data) {
+                scope.collateral = data;
+                console.log(data);
+                scope.loanTransactions = scope.collateral.loanTransactionData;
+                for (var i=0; i<scope.loanTransactions.length; i++){
+                	scope.loanTransactions[i].lastRepaymentDate.date = new Date(scope.loanTransactions[i].lastRepaymentDate.date);
+                }
+                console.log(scope.loanTransactions)
```

Comment:
I think this must be a temporary `console.log` statement. Please remove it.

Reply:
Removed unwanted console logs